src/assets/styles/PrimaryButton.css

```diff
@@ -26,4 +72,40 @@
 
 #submitButton {
   width: 80%;
+}
+
+@media (min-width: 768px) {
+  .registerForm{
+    width: 60%
+  } 
+}
+
+@media (min-width: 1024px) {
```

Comment:
Media query 768 and 1024 are duplicated use one

Reply:
Thanks leonel. I refactor the mediaqueries duplication. No need for that much code to obtain same results.